packages/services/server/src/environment.ts

```diff
@@ -92,6 +99,7 @@ const RedisModel = zod.object({
   REDIS_HOST: zod.string(),
   REDIS_PORT: NumberFromString,
   REDIS_PASSWORD: emptyString(zod.string().optional()),
+  REDIS_TLS_ENABLED: BooleanFromString,
```

Comment:
This new environment variable is not documented within `packages/services/server/README.md`.

Reply:
added